website/views.py

```diff
@@ -558,7 +558,10 @@ def get_initial(self):
     def post(self, request, *args, **kwargs):
         
         # resolve domain
-        url = request.POST.get("url").replace("www.","https://")
+        url = request.POST.get("url")
```

Comment:
We need to remove www and http for all domains so they are stored the same in the db.

Reply:
resolved